mozilla_django_oidc/middleware.py

```diff
@@ -0,0 +1,92 @@
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
+
+import time
+
+import django
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+from django.utils.crypto import get_random_string
+
+from mozilla_django_oidc.utils import absolutify, import_from_settings
+
+
+# Django 1.10 makes changes to how middleware work. In Django 1.10+, we want to
+# use the mixin so that our middleware works as is.
+if django.VERSION >= (1, 10):
+    from django.utils.deprecation import MiddlewareMixin
+else:
+    class MiddlewareMixin(object):
+        pass
+
+
+OIDC_URLS = [
+    reverse(name)
+    for name in [
+        'oidc_authentication_init',
+        'oidc_authentication_callback',
+        'oidc_logout'
+    ]
```

Comment:
I was thinking the latter. Having a config entry that defaults to an empty list and we always append these 3 entries.